microsoft-edge/devtools-guide-chromium/storage/cookies.md

```diff
@@ -30,16 +30,15 @@ See [Using HTTP cookies](https://developer.mozilla.org/docs/Web/HTTP/Cookies).
 <!-- ====================================================================== -->
 ## Open the Cookies pane
 
-1. [Open DevTools](/microsoft-edge/devtools-guide-chromium/open).
+1. Open DevTools on the webpage you want to edit. To open DevTools, use one of the techniques described in [Open DevTools](/microsoft-edge/devtools-guide-chromium/open).
 
-1. Click the **Application** tab to open the **Application** panel.  The **Manifest** pane opens:
+2. Click the **Application** icon (![Application icon in Activity Bar](images/application-storage-cookies-application-icon.png)) in the Activity Bar or select "Application" in the Quick View dropdown list to open the **Application** panel.  The **Manifest** pane opens:
```

Comment:
"icon in Activity Bar" - pls check to see if wording & captures should show Focus Mode UI or not

Reply:
@dan-wesley pls close this thread is done/feasible.